### ISSUE TITLE
"Show More" apparently not working.

### ISSUE THREAD

**Issue by justpip** [rugantio/fbcrawl#69]
Mbasic.facebook's "Show More" does not show more posts. As a result the fbcrawl outputs the same, most recent, posts over and over.

**Comment by natsinger** [comment]
Hey try to change the div count( from 1 to 2) in :
"#new_page is different for groups
        if self.group == 1:
            new_page = response.xpath("//div[contains(@id,'stories_container')]/div[2]/a/@href").extract()"

to :
if self.group == 1:
            new_page = response.xpath("//div[contains(@id,'stories_container')]/div[1]/a/@href").extract()

**Comment by cavdimio** [comment]
Above solution doesn't work for me

**Comment by lijia-ma** [comment]
> Mbasic.facebook's "Show More" does not show more posts. As a result the fbcrawl outputs the same, most recent, posts over and over.

I think this is exactly why fbcrawl doesn't work anymore.

**Comment by Rahulsunny11** [comment]
did anyone found solution to this?

**Comment by gonzalo-gongora** [comment]
Did anyone find the solution to this? x2

I tried browsing different pages in `mbasic.facebook.com` mode and the `Show More` button doesn't seem to work. 
It may be a bug within the mbasic page, can anyone confirm?

**Comment by Rahulsunny11** [comment]
> Did anyone find the solution to this? x2
> 
> I tried browsing different pages in `mbasic.facebook.com` mode and the `Show More` button doesn't seem to work.
> It may be a bug within the mbasic page, can anyone confirm?

yes gonzalo-gongora, there is an issue with mbasi.facebook.com.. show more in mbasic version is not working
